### PR TITLE
initial implementation

### PR DESCRIPTION
**Pull request by morethanreal**
@cdata @notwaldorf PTAL

### CODE REVIEW

**Comment by morethanreal** [comment]
plz hold off on reviewing. some changes coming

**Comment by morethanreal** [comment]
@cdata PTAnotherL? Will update `bower.json` when PolymerElements/paper-dialog-scrollable#1 gets in.

File: demo/index.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<!doctype html>
+<!--
+  @license
```
Comment (notwaldorf) [inline demo/index.html]: nit: no indenting :)

File: paper-dialog-common.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+*/
+
+:host {
+  display: block;
+  margin: 24px 40px;
+
+  background: var(--paper-dialog-background-color);
```
Comment (notwaldorf) [inline paper-dialog-common.css]: I couldn't see if these were documented anywhere. How would an implementer of this know what variables to define?

File: demo/simple-dialog.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+<script>
+
+(function() {
+
+  Polymer({
+
+    is: 'simple-dialog',
+
+    enableCustomStyleProperties: true,
```
Comment (cdata) [inline demo/simple-dialog.html]: This can be removed as of 0.9

File: index.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<!--
+  @license
+  Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+  This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+  The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+  The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+  Code distributed by Google as part of the polymer project is also
+  subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
```
Comment (cdata) [inline index.html]: Indentation

File: paper-dialog-common.css (hunk 2 above)
Comment (cdata) [inline paper-dialog-common.css]: Hopefully `iron-doc-viewer` will eventually expose this information.

File: test/test-dialog.html [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+<script>
+
+(function() {
+
+  Polymer({
+
+    is: 'test-dialog',
+
+    enableCustomStyleProperties: true,
```
Comment (cdata) [inline test/test-dialog.html]: This can be removed as of 0.9

File: bower.json [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/PolymerElements/paper-dialog-behavior"
+  },
+  "license": "MIT",
+  "homepage": "https://github.com/PolymerElements/paper-dialog-behavior",
+  "ignore": [],
+  "dependencies": {
+    "polymer": "Polymer/polymer#v0.8.0-rc.7",
```
Comment (cdata) [inline bower.json]: `v0.9.0-rc.1` has landed :tada:

File: demo/simple-dialog.html (hunk 3 above)
Comment (morethanreal) [inline demo/simple-dialog.html]: Fixed.

File: index.html (hunk 4 above)
Comment (morethanreal) [inline index.html]: Fixed.

File: paper-dialog-common.css (hunk 2 above)
Comment (morethanreal) [inline paper-dialog-common.css]: They should be included in the doc section. I will do that later.

File: test/test-dialog.html (hunk 5 above)
Comment (morethanreal) [inline test/test-dialog.html]: Fixed.

File: demo/index.html (hunk 1 above)
Comment (morethanreal) [inline demo/index.html]: Fixed.

File: bower.json (hunk 6 above)
Comment (morethanreal) [inline bower.json]: Fixed.

**Comment by morethanreal** [comment]
Updated to 0.9. PTAnotherL?

**Comment by morethanreal** [comment]
Ping?

**Comment by notwaldorf** [comment]
I think this lgtm (maybe squash all the things too?), but @cdata should double check since it was his review :stuck_out_tongue:

File: bower.json [hunk 7]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "paper-dialog-behavior",
+  "version": "0.9.0",
+  "authors": "The Polymer Authors",
```
Comment (cdata) [inline bower.json]: During the meeting today it was revealed that the element catalogue is parsing `bower.json` and using the `"description"` field for some things :memo:

File: index.html (hunk 4 above)
Comment (cdata) [inline index.html]: If it is fixed, why is it still indented? :confused:

**Comment by cdata** [comment]
Minor formatting feedback, otherwise LGTM :+1:

**Pull request merged**